### ISSUE TITLE
Dashboard shows multiple instance for the same PodDisruptionBudget, with different results

### ISSUE THREAD

**Issue by sbkg0002** [FairwindsOps/polaris#583]
### Installation Process
 
Installed via the latest Helm chart onto AKS version 1.19.
### Polaris Version

```
/opt/app $ polaris version
Polaris version:4.0.4
```

### Expected Behavior

A single PodDisruptionBudget, that shows ` disruptionsAllowed is greater than zero` / all OK.

### Actual Behavior

Multiple instances of the same `PodDisruptionBudget`, with various results:

<img width="843" alt="Screenshot 2021-07-08 at 09 56 32" src="https://user-images.githubusercontent.com/1254233/124884855-f9ae2d80-dfd2-11eb-9656-23e4c1058810.png">

### Steps to Reproduce

```polaris --audit --output-format score```

Will output:
```json
...
    {
      "Name": "coredns-pdb",
      "Namespace": "kube-system",
      "Kind": "PodDisruptionBudget",
      "Results": {
        "pdbDisruptionsIsZero": {
          "ID": "pdbDisruptionsIsZero",
          "Message": "disruptionsAllowed is not greater than zero",
          "Details": null,
          "Success": false,
          "Severity": "warning",
          "Category": "Reliability"
        }
      },
      "PodResult": null,
      "CreatedTime": "0001-01-01T00:00:00Z"
    },
    {
      "Name": "coredns-pdb",
      "Namespace": "kube-system",
      "Kind": "PodDisruptionBudget",
      "Results": {
        "pdbDisruptionsIsZero": {
          "ID": "pdbDisruptionsIsZero",
          "Message": "disruptionsAllowed is greater than zero",
          "Details": null,
          "Success": true,
          "Severity": "warning",
          "Category": "Reliability"
        }
      },
      "PodResult": null,
      "CreatedTime": "0001-01-01T00:00:00Z"
    },
...
```

### Additional Context

**The PodDisruptionBudget**:
```yaml
apiVersion: policy/v1beta1
kind: PodDisruptionBudget
metadata:
  annotations: {}
  generation: 1
  labels:
    addonmanager.kubernetes.io/mode: Reconcile
  name: coredns-pdb
  namespace: kube-system
spec:
  minAvailable: 1
  selector:
    matchLabels:
      k8s-app: kube-dns
```
<details>
  <summary>The deployment Spec</summary>

```yaml
apiVersion: apps/v1
kind: Deployment
metadata:
  annotations:
    deployment.kubernetes.io/revision: '4'
  generation: 37
  labels:
    addonmanager.kubernetes.io/mode: Reconcile
    k8s-app: kube-dns
    kubernetes.io/cluster-service: 'true'
    kubernetes.io/name: CoreDNS
    version: v20
  name: coredns
  namespace: kube-system
spec:
  progressDeadlineSeconds: 2147483647
  replicas: 3
  revisionHistoryLimit: 2
  selector:
    matchLabels:
      k8s-app: kube-dns
      version: v20
  strategy: [...]
  template:
    metadata:
      annotations:
        prometheus.io/port: '9153'
      labels:
        k8s-app: kube-dns
        kubernetes.io/cluster-service: 'true'
        version: v20
    spec:
      affinity:
        nodeAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
            - preference:
                matchExpressions:
                  - key: kubernetes.azure.com/mode
                    operator: In
                    values:
                      - system
              weight: 100
        podAntiAffinity:
          preferredDuringSchedulingIgnoredDuringExecution:
            - podAffinityTerm:
                labelSelector:
                  matchExpressions:
                    - key: k8s-app
                      operator: In
                      values:
                        - kube-dns
                topologyKey: failure-domain.beta.kubernetes.io/zone
              weight: 10
            - podAffinityTerm:
                labelSelector:
                  matchExpressions:
                    - key: k8s-app
                      operator: In
                      values:
                        - kube-dns
                topologyKey: kubernetes.io/hostname
              weight: 5
      containers: [...]
      dnsPolicy: Default
      nodeSelector:
        beta.kubernetes.io/os: linux
      priorityClassName: system-node-critical
      restartPolicy: Always
      schedulerName: default-scheduler
      securityContext: {}
      serviceAccount: coredns
      serviceAccountName: coredns
      terminationGracePeriodSeconds: 30
      tolerations: [...]
      volumes: [...]
```

</details>

### References

Not that I know off.

**Comment by fsz-codeshop** [comment]
Mine is showing multiple instaces for Polaris ingress
![image](https://user-images.githubusercontent.com/79223916/124980146-67c21700-e00a-11eb-9425-8e40f811ef26.png)

**Comment by rbren** [comment]
@fsz-codeshop @sbkg0002 this should be fixed in 4.0.5 - can you verify?

**Comment by fsz-codeshop** [comment]
> @fsz-codeshop @sbkg0002 this should be fixed in 4.0.5 - can you verify?

Yeap, working fine here. Thanks @rbren :D :rocket:

**Comment by sbkg0002** [comment]
Can also confirm, thanks rbren! Can be closed!
